Libraries/DeviceInfo/DeviceInfo.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DeviceInfo
+ */
+'use strict';
+
+var RCTDeviceInfo = require('NativeModules').DeviceInfo;
+
+/**
+ * `DeviceInfo` gives you information about your running application and the
+ * device you are running on.
+ *
+ * #### Device Name on Android
+ *
+ * If you want to get the device name on Android then you will need to add the
+ * bluetooth permission to your application's `AndroidManifest.xml` file.
+ *
+ * ```
+ * <uses-permission android:name="android.permission.BLUETOOTH"/>
+ * ```
+ */
+var DeviceInfo = {
+  /**
+   *  Return a unique ID for the current device (this is the IDFV on iOS so it will change if all apps from the current apps vendor have been previously uninstalled)
+   */
+  get UniqueID() { return RCTDeviceInfo.uniqueId; },
+  /**
+   *  Return a system identifier for the type of device e.g. iPhone7,2 or the board on Android e.g. goldfish
+   */
+  get DeviceID() { return RCTDeviceInfo.deviceId; },
+  /**
+   *  Return the device manufacturer e.g. Apple
+   */
+  get Manufacturer() { return RCTDeviceInfo.systemManufacturer; },
+  /**
+   *  Return a user friendly device model e.g. iPad Air 2
+   */
+  get Model() { return RCTDeviceInfo.model; },
+  /**
+   *  Return the name of the operating system running on the device e.g. iPhone OS
+   */
+  get SystemName() { return RCTDeviceInfo.systemName; },
+  /**
+   *  Return the version of the operating system running on the device e.g. 9.0
+   */
+  get SystemVersion() { return RCTDeviceInfo.systemVersion; },
+  /**
+   *  Return the application's bundle identifier e.g. com.facebook.internal.uiexplorer.local
+   */
+  get BundleID() { return RCTDeviceInfo.bundleId; },
```

Comment:
`BundleID` is a very easy to confuse with the JavaScript bundle. We should change it to `PackageName`